CoreStubs/ArchiveControl/module.php

```diff
@@ -23,6 +24,25 @@ private function SetVariableData($VariableID, $Data)
         $this->Archive[$VariableID] = $Data;
     }
 
+    public function StubsAddAggregatedValues(int $VariableID, int $AggregationLevel, array $AggregationData)
+    {
+        usort($AggregationData, function ($a, $b)
+        {
+            return $a['TimeStamp'] <=> $b['TimeStamp'];
+        });
+
+        if (isset($this->AggregatedArchive[$VariableID][$AggregationLevel])) {
+            $AggregatedArchiveData = $this->AggregatedArchive[$VariableID][$AggregationLevel];
+            if ((count($AggregatedArchiveData) > 0) && (count($AggregationData) > 0) &&
+                ($AggregationData[0]['TimeStamp'] < $AggregatedArchiveData[count($AggregatedArchiveData) - 1]['TimeStamp'])) {
+                throw new Exception('It is not yet possible to addaggregated  values before the newest');
+            }
+            $this->AggregatedArchive[$VariableID][$AggregationLevel] = array_merge($AggregatedArchiveData, $AggregationData);
```

Comment:
Schick. Sollten wir in AddLoggedValues ebenfalls "verbessern", damit es konsistent